subsys/nrf_security/src/drivers/ironside/ironside_psa.h

```diff
@@ -0,0 +1,97 @@
+/*
+ * Copyright (c) 2025 Nordic Semiconductor ASA
+ * SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+ */
+
+#ifndef IRONSIDE_PSA_H
+#define IRONSIDE_PSA_H
+
+#include <psa/crypto.h>
+
+/* The following header must be provided externally and with these types:
+ * - ironside_psa_pake_operation_t
+ */
+#include "ironside_psa_types.h"
+
+psa_status_t ironside_psa_get_key_slot(mbedtls_svc_key_id_t key_id, psa_key_lifetime_t *lifetime,
```

Comment:
Should we provide a rationale for why this interface is needed here? We have quite specific reasons for why we need to have this interface in place, and IMO it would be good to describe it publicly.

Reply:
Added. This may be useful for other reviewers to read as well.